docs/troubleshooting-faq/faq.md

```diff
@@ -0,0 +1,101 @@
+1. The sample or project ID(s) that you specified during download do(es) not exist.
+
+Therefore, you may wish to list some of the files in the data release S3 bucket to confirm the sample/project files you are trying to download actually exist.
+
+Data files in each release are organized on S3 as:
+```{ .console .no-copy title="Release file structure"}
+{Release}
+    ├── {Project ID}
+    │   └── {Sample ID}
+    │       └── {Library files}
+    ├── bulk_metadata.tsv (if applicable)
+    ├── bulk_quant.tsv (if applicable)
+    └── single_cell_metadata.tsv
+```
+
+To list all available files, you'll need to use the AWS CLI [in a terminal](../software-platforms/general-tools/using-the-terminal.md) and run `aws s3 ls` to list the contents of a given S3 bucket.
+Don't forget to [log into your AWS `openscpca` profile first](../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session)!
+
+
+1. First, find the names of all the data releases, which are named based on their release date in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601).
+You can use the [`download-data.py`](../getting-started/accessing-resources/getting-access-to-data.md#download-data-structure) script with the `--list-releases` flag to do this:
+
+    ```bash
+    # Ensure you are in the top-level of the repository
+    # cd path/to/OpenScPCA-analysis
+
+    # List all releases in the data release bucket
+    # If you're _not_ working on Lightsail for Research, specify your AWS profile name with `--profile openscpca`
+    ./download-data.py --list-releases --profile openscpca
+    ```
+
+1. Identify the most recent date from the output.
+In the example output below, the most recent release is `2024-05-01`.
+    ```{ .console .no-copy title="Output from listing all releases"}
+    Available release dates:
+    2024-03-11
+    2024-04-06
+    2024-05-01
+    ```
+
+1. All data is stored on S3 in `s3://openscpca-data-release/{release name}/`.
+Therefore, you can list files in a given release as, for example,
+    ```bash
+    # List all releases in the 2024-05-01 release
+    # If you're _not_ working on Lightsail for Research, specify your AWS profile name with `--profile openscpca`
+    aws s3 ls s3://openscpca-data-release/2024-05-01/ --profile openscpca
+    ```
+
+    This has the (abbreviated) output:
+    ```{ .console .no-copy title="Output from listing all projects in the 2024-05-01 release"}
+    PRE SCPCP000001/
+    PRE SCPCP000002/
+    PRE SCPCP000003/
+    PRE SCPCP000004/
+    ```
+
+    Note that the `PRE` prefix appears for directories that are inside the overall data release bucket, `s3://openscpca-data-release`.
+
+    !!! tip "Don't forget the trailing slash!"
+        The trailing slash at the end of `s3://openscpca-data-release/2024-05-01/` is necessary for contents to be listed.
```

Comment:
Just noting that you can also use `--recursive` but that will show all files in the directory, which we probably don't want them to do for entire releases.